msi/release.sh

```diff
@@ -1,17 +1,56 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+me=$(basename $0)
+function showUsage() {
+	cat <<-EOF
 
-test -z "$1" && {
-	echo "Usage: $0 <version> [optional components]"
-	exit 1
+Usage: msi/$me [-o <dir>] <version> [<optional-package>...]
```

Comment:
OK, you talked me into it. :wink: Yeah, you're probably right that that is better.